pkg/nuclide/ocaml-base/lib/MerlinProcess.js

```diff
@@ -243,8 +257,14 @@ export async function getInstance(file: NuclideUri): Promise<?MerlinProcess> {
 
   const dotMerlinPath = await findNearestFile('.merlin', file);
 
+  opamBin = await getOpamBin();
+  env = process.env.PATH;
+  if (opamBin) {
+    env = env + ":" + opamBin
```

Comment:
They can be joined like strings here, but the platform-specific delimiter should be used from Node's [`path.delimiter`](https://nodejs.org/api/path.html#path_path_delimiter):

```js
import path from 'path';

...

if (opamBin) {
  env = `${env}${path.delimiter}${opamBin}`;
}
```